src/sentry/auth_v2/endpoints/auth_merge_user_accounts.py

```diff
@@ -14,9 +14,9 @@
 
 
 class AuthMergeUserAccountsValidator(serializers.Serializer):
```

Comment:
You can make `AuthMergeUserAccountsValidator` subclass `CamelSnakeSerializer` and keep the snake case here, we like snake case on the backend